app/src/main/java/com/asfoundation/wallet/my_wallets/more/MoreDialogViewModel.kt

```diff
@@ -24,8 +37,46 @@ class MoreDialogViewModel(data: MoreDialogData) :
       data.totalFiatBalance,
       data.appcoinsBalance,
       data.creditsBalance,
-      data.ethereumBalance,
-      data.showDeleteWallet
+      data.ethereumBalance
```

Comment:
Ideally we should replace this with `SavedStateHandle`, like we have for example in the `RecoverPasswordViewModel`.
This makes it possible to remove the `MoreDialogData`, `MoreDialogModule` and `MoreDIalogViewModelFactory` without changing any of the functionality

Reply:
Refactored